src/main/java/seedu/address/model/util/SampleDataUtil.java

```diff
@@ -18,25 +19,25 @@
  */
 public class SampleDataUtil {
     public static Person[] getSamplePersons() {
-        return new Person[] {
+        return new Person[]{
             new Person(new Name("Alex Yeoh"), new Phone("87438807"), new Email("alexyeoh@example.com"),
-                new Address("Blk 30 Geylang Street 29, #06-40"),
-                getTagSet("friends")),
+                        new Address("Blk 30 Geylang Street 29, #06-40"),
```

Comment:
Perhaps this should be 8 spaces relative to line 22? But not sure why the original AB3 passed checkstyle
Similar concept for the rest of the lines in this file

Reply:
Changing this will lead to a checkstyle error tho.

![image](https://user-images.githubusercontent.com/62068282/95715783-ff255400-0c9c-11eb-8547-40e620831309.png)
This is why I went to change the indentation

Reply:
Alright got it